### PR TITLE
POC - complete app structure

### PR DESCRIPTION
**Pull request by shlomitc**
I think it's super important that we allow our consumers to have their complete page layout with minimal effort.

Tell me what you think about it

### CODE REVIEW

File: src/Structure/index.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
```
Comment (argshook) [inline src/Structure/index.js]: this shouldn't live under `src` cause `import-path` will make it importable which may be mistaken for a real component.

unless that's by intention?

**Comment by shlomitc** [comment]
By intention, to make it easy to create a structure.
Or do you think it should be just guidelines?

File: src/Structure/index.js (hunk 1 above)
Comment (shlomitc) [reply src/Structure/index.js]: By intention, to make it easy to create a structure.
Or do you think it should be just guidelines?

**Comment by shlomitc** [comment]
closing it until will get some time to finish it correctly

**Pull request closed without merging**